Automattic/WP-Job-Manager
# Add custom fields for company facebook and linkedin page

Fixes https://github.com/Automattic/WP-Job-Manager/issues/1821

Backend Fields:
![wp-job-manager-backend-fields](https://user-images.githubusercontent.com/26354653/62148834-1ab27200-b318-11e9-82ee-fca33d566255.png)

Frontend Fields:
![frontend-fields](https://user-images.githubusercontent.com/26354653/62148882-2e5dd880-b318-11e9-83f9-741b8b1afd86.png)


#### Changes proposed in this Pull Request:

* Add field for company Facebook and LinkedIn Page link.

#### Testing instructions:

* Add new Job from frontend and backend and fill out `Company Facebook` and `Company LinkedIn` fields with the respective page ID.
* On single Job page (frontend) check Facebook and LinkedIn links.
* Link should display if Facebook and LinkedIn page ID provided.


#### Proposed changelog entry for your changes:

##### Enhancement 
* Added fields for Company Facebook and LinkedIn page.
